arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -2008,3 +2016,7 @@ void RocksDBEngine::releaseTick(TRI_voc_tick_t tick) {
 }
 
 }  // namespace arangodb
+
+// -----------------------------------------------------------------------------
+// --SECTION--                                                       END-OF-FILE
+// -----------------------------------------------------------------------------
```

Comment:
Okay, can you go through and remove these at some point? It seems like we don't need an extra three lines of boilerplate at the end of every file to tell us that we're at the end of the file. Purely redundant.